src/axom/config.hpp.in

```diff
@@ -31,6 +31,12 @@
 #define AXOM_BIN_DIR "@AXOM_BIN_DIR@"
 #cmakedefine AXOM_DATA_DIR "@AXOM_DATA_DIR@"
 
+/*
+ * Indicates whether Axom is compiled with Annotations that can facilitate
+ * evaluation of performance with various performance tools.
+ */
+#cmakedefine AXOM_ENABLE_ANNOTATIONS
```

Comment:
For consistency, please change this to ``AXOM_USE_ANNOTATIONS``.

We've been using ``AXOM_ENABLE_*`` for build system features and ``AXOM_USE_*`` in the code.

Reply:
Done. See d5657e8